seed/utils/geocode.py

```diff
@@ -0,0 +1,116 @@
+# !/usr/bin/env python
+# encoding: utf-8
+
+import requests
+
+from django.conf import settings
+
+from django.contrib.gis.geos import GEOSGeometry
+
+
+def long_lat_wkt(state):
+    if state.long_lat:
+        return GEOSGeometry(state.long_lat,srid=4326).wkt
```

Comment:
@adrian-lara what does srid=4326 mean?

Reply:
The Spatial Reference Identifier references which coordinate system to use. What I found from my research was that 4326 is pretty common around the world. It's used for GPS and is also the default for PostGIS and GeoJSON.

Reply:
@adrian-lara ok that's fine then. Worth an inline comment for that then (maybe not even a bad idea to include a url link to the data you used to find it or something). Just so if someone if reading the code a year from now and they're like what's this hard coded number, maybe that's my problem! lol